content/200-concepts/100-components/01-prisma-schema/02-data-sources.mdx

```diff
@@ -39,3 +39,9 @@ datasource db {
   url      = "postgresql://johndoe:mypassword@localhost:5432/mydb?schema=public&sslmode=require&sslcert=../server-ca.pem&sslidentity=../client-identity.p12&sslpassword=<REDACTED>"
 }
 ```
+
+## TLS certificates
+
+Some data source providers, such as Microsoft SQL Server, enable you to configure your connection with TLS by providing parameters in the connection `url`.
+
+- [Configure a TLS connection with Microsoft SQL Server](../../database-connectors/sql-server#connection-details)
```

Comment:
I am not sure TLS and SSL are different concepts on a database level here - most often this refers to the same thing.

Reply:
Yep, but it would be linking to a bit about TLS from a section specifically about SSL. I thought it would look odd. Maybe the user would assume the link pointed to SSL info and then they click and its TLS. 

That's how it played out in my head. Do you think it would be ok to just link to the TLS section from the SSL section? Maybe im over thinking it....

Reply:
We could also renamed the headline to "securing the database connection" or something like that. But my vocabulary in that area is not very refined, so there might be better terms to describe what unifies that.

Reply:
Thats a good idea. It would require checking for anything linking to that anchor. I couldnt control external links, but thats the danger with changing anchor links.

Reply:
Oh, there _are_ solutions for that problem: https://github.com/prisma/docs/issues/1680 Redirects for anchors basically.

But I think in this case this should be fine without. This is not a prime target for external links and this is a short page anyway.